Makefile

```diff
@@ -44,6 +44,12 @@ IMPORT_LOG=.import.log
 
 GIT_SHA=$(shell git rev-parse HEAD)
 GIT_CLOSEST_TAG=$(shell git describe --abbrev=0 --tags)
+ifeq ($(GIT_CLOSEST_TAG), main)
+	GIT_CLOSEST_TAG=v0.0.0
```

Comment:
I think we should fail instead. I don't even know how this 'main' tag ended up being created, but it certainly messed up the release process. Having a default like v0.0.0 means that that process might have succeeded and created v0.0.0 release - not at all what we want.

Reply:
I'm actually having second thoughts about it. I was about to commit this:
```diff
+semver_regex := ^v\d+\.\d+\.\d+$$
 GIT_SHA=$(shell git rev-parse HEAD)
 GIT_CLOSEST_TAG=$(shell git describe --abbrev=0 --tags)
+ifneq ($(GIT_CLOSEST_TAG),$(shell echo ${GIT_CLOSEST_TAG} | grep -E "$(semver_regex)"))
+       $(error GIT_CLOSEST_TAG=$(GIT_CLOSEST_TAG) is not in the semver format $(semver_regex))
+endif
```
but this would break the whole Makefile in case there are some custom tags. I don't think the Makefile entry is the place for such check, it should be done only in places where having a valid semver is actually necessary.